css/index.css

```diff
@@ -111,7 +111,56 @@ p {
     align-items: center;
     justify-content: center;
 }
+.boxes .box:nth-child(1) {
+    background: teal;
 
+
```

Comment:
Good to see you targeted the boxes using css selectors rather than creating a brand new class, this shows you have a great understanding of css